hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -381,9 +381,10 @@ public final class HoodieMetadataConfig extends HoodieConfig {
 
   public static final ConfigProperty<Boolean> ENABLE_METADATA_INDEX_PARTITION_STATS = ConfigProperty
       .key(METADATA_PREFIX + ".index.partition.stats.enable")
-      .defaultValue(false)
+      .defaultValue(true)
```

Comment:
we do this at per engine level. thats why we did not flip this value. 

if you check HoodieMetadataConfig, 
we have getDefaultColStatsEnable() which will choose the right value for col stats for different engines. 
and the default value for partitions stats enable goes along w/ the derived value for col stats. 
if col stats is enabled, PSI will also be enabled. if not, it will not. 

We just need to update our release docs to call this out.

Reply:
I have updated the description ,PTAL